scripts/run_changed.js

```diff
@@ -52,35 +52,82 @@ const alwaysRunTestPaths = [
   'integration/webpack'
 ];
 
+/**
+ * These files trigger tests in other dirs
+ */
+const specialPaths = {
+  'scripts/emulator-testing/emulators/firestore-emulator.ts': ['packages/firestore'],
+  'scripts/emulator-testing/emulators/database-emulator.ts': ['packages/database'],
+  'scripts/emulator-testing/emulators/emulator.ts': ['packages/firestore', 'packages/database'],
+  'scripts/emulator-testing/firestore-test-runner.ts': ['packages/firestore'],
+  'scripts/emulator-testing/database-test-runner.ts': ['packages/database']
+};
+
 /**
  * Identify modified packages that require tests.
  */
 async function getChangedPackages() {
+  const packageInfo = JSON.parse(
+    (await exec('npx lerna ls --json', { cwd: root })).stdout
```

Comment:
Can use the already installed lerna at "node_modules/.bin/lerna" to remove the networking.

Reply:
npx will run the local installed version if available.